testing/utils/gather-extra.sh

```diff
@@ -115,79 +114,79 @@ queue ${ARTIFACT_DIR}/infrastructure.yaml $OC get infrastructure/cluster
-        FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/audit.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false --path=audit/audit.log
-    done < /tmp/nodes
+	while IFS= read -r i; do
+		mkdir -p ${ARTIFACT_DIR}/nodes/$i
+		queue ${ARTIFACT_DIR}/nodes/$i/heap oc --insecure-skip-tls-verify get --request-timeout=20s --raw /api/v1/nodes/$i/proxy/debug/pprof/heap
+		FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/journal.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false
+		FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/journal-previous.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false --boot=-1
+		FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/audit.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false --path=audit/audit.log
+	done </tmp/nodes
 fi
 
 # gather nodes first in parallel since they may contain the most relevant debugging info
 while IFS= read -r i; do
-  mkdir -p ${ARTIFACT_DIR}/nodes/$i
-  FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/kube-apiserver.audit.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false --path=kube-apiserver/audit.log
-done < /tmp/control_plan_nodes
+	mkdir -p ${ARTIFACT_DIR}/nodes/$i
+	FILTER=gzip queue ${ARTIFACT_DIR}/nodes/$i/kube-apiserver.audit.gz oc --insecure-skip-tls-verify adm node-logs $i --unify=false --path=kube-apiserver/audit.log
+done </tmp/control_plan_nodes
 
 echo "INFO: gathering the audit logs for each master"
 paths=(openshift-apiserver kube-apiserver oauth-apiserver etcd)
-for path in "${paths[@]}" ; do
-  if [[ "$platform_type" == "BareMetal" ]]; then
-      echo "WARNING: gathering of audit logs on the master disabled on baremetal platforms."
-      continue
-  fi
-  output_dir="${ARTIFACT_DIR}/audit_logs/$path"
-  mkdir -p "$output_dir"
-
-  # Skip downloading of .terminating and .lock files.
-  oc adm node-logs --role=master --path="$path" | \
-    grep -v ".terminating" | \
-    grep -v ".lock" | \
-  tee "${output_dir}.audit_logs_listing"
-
-  # The ${output_dir}.audit_logs_listing file contains lines with the node and filename
-  # separated by a space.
-  while IFS= read -r item; do
-    node=$(echo $item |cut -d ' ' -f 1)
-    fname=$(echo $item |cut -d ' ' -f 2)
-    echo "INFO: Queueing download/gzip of ${path}/${fname} from ${node}";
-    echo "INFO:   gziping to ${output_dir}/${node}-${fname}.gz";
-    FILTER=gzip queue ${output_dir}/${node}-${fname}.gz oc --insecure-skip-tls-verify adm node-logs ${node} --path=${path}/${fname}
-  done < ${output_dir}.audit_logs_listing
+for path in "${paths[@]}"; do
+	if [[ "$platform_type" == "BareMetal" ]]; then
+		echo "WARNING: gathering of audit logs on the master disabled on baremetal platforms."
+		continue
+	fi
+	output_dir="${ARTIFACT_DIR}/audit_logs/$path"
+	mkdir -p "$output_dir"
+
+	# Skip downloading of .terminating and .lock files.
+	oc adm node-logs --role=master --path="$path" |
+		grep -v ".terminating" |
+		grep -v ".lock" |
```

Comment:
@drewrip , this looks wrong to me, the trailing `\` is necessary
can you revert the changes in this file? if I'm not missing anything, they're only cosmetic, and this files comes from OpenShift CI code, so better stay as close to them as possible, even if that won't change much of anything 🙃

Reply:
Gotcha! Thanks for pointing this out. Should be fixed.